hedera-mirror-rosetta/build/Dockerfile

```
@@ -2,19 +2,21 @@
 # Importer, Rosetta and PostgreSQL into one image
 # and run the services using supervisord
 
-# ------------------------------  Rosetta  ------------------------------- #
-FROM golang:1.17.1 as rosetta-builder
-COPY hedera-mirror-rosetta /hedera-mirror-rosetta
-WORKDIR /hedera-mirror-rosetta
-ARG VERSION=development
-RUN go build -ldflags="-w -s -X main.Version=${VERSION}" -o hedera-mirror-rosetta
-
-# ---------------------------- Importer ----------------------------- #
-FROM openjdk:11.0 as java-builder
-
-RUN apt-get update
-COPY . ./hedera-mirror-node
-RUN cd hedera-mirror-node && ./mvnw --batch-mode --no-transfer-progress --show-version clean package -DskipTests -pl hedera-mirror-importer
+FROM ubuntu:20.04 as builder
+ARG DEBIAN_FRONTEND=noninteractive
+# GIT_REF can be a branch, a commit SHA, or a tag
+ARG GIT_REF=main
+RUN apt-get update && apt-get install -y gcc git openjdk-11-jdk-headless
+RUN git clone https://github.com/hashgraph/hedera-mirror-node && cd hedera-mirror-node && git checkout ${GIT_REF}
```

Do a shallow clone to improve performance
```suggestion
RUN git clone -–depth 1 https://github.com/hashgraph/hedera-mirror-node && cd hedera-mirror-node && git checkout ${GIT_REF}
```
```suggestion
RUN git clone https://github.com/hashgraph/hedera-mirror-node && cd hedera-mirror-node && git checkout ${GIT_REF}
```